src/parquet/parquet.h

```diff
@@ -40,6 +40,26 @@
 #include <thrift/protocol/TBinaryProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
 
+//macros to disable padding. Required for Int96
```

Comment:
I don't think we should give an option of disabling padding until we specialize the templates to handle padding as well.